core/pom.xml

```diff
@@ -35,6 +35,16 @@
   <url>http://spark.apache.org/</url>
   <dependencies>
     <dependency>
+      <groupId>net.sf.py4j</groupId>
+      <artifactId>py4j</artifactId>
+      <version>0.8.2.1</version>
+    </dependency>
```

Comment:
is there any reason to move this?  if not, leave it in the old location, so its easier to understand where it got added w/ git blame etc.